src/base.ts

```diff
@@ -27,11 +27,6 @@ export interface IResource extends IConstruct {
    */
   readonly apiVersion: string;
 
-  /**
-   * The group portion of the API version (e.g. "authorization.k8s.io")
-   */
-  readonly apiGroup: string;
-
```

Comment:
Removed because its now a duplicate of the same [property](https://github.com/cdk8s-team/cdk8s-plus/blob/k8s-25/main/src/api-resource.generated.ts#L10) in `IApiResource`.